Source/WebCore/workers/WorkerScriptLoader.h

```diff
@@ -61,9 +61,10 @@ class WorkerScriptLoader final : public RefCounted<WorkerScriptLoader>, public T
     WTF_MAKE_TZONE_ALLOCATED(WorkerScriptLoader);
     WTF_OVERRIDE_DELETE_FOR_CHECKED_PTR(WorkerScriptLoader);
 public:
-    static Ref<WorkerScriptLoader> create()
+    enum class AlwaysUseUTF8 : bool { No, Yes };
```

Comment:
Nit: Is there a reason this isn't the same enum as `WorkerScriptLoader::AlwaysUseUTF8`?

Reply:
I'm a bit confused. This is `WorkerScriptLoader::AlwaysUseUTF8`.

Reply:
Sorry `ShouldDecodeAsUTF8Only`...